src/main/resources/category/java/common.xml

```diff
@@ -855,9 +855,9 @@ class Foo {
     (: if methods are chained, only report 1 in the chain :)
     and not(ancestor::MethodCall[ArgumentList[@Size>0]]/MethodCall)
   ]
-  (: except if method name starts with 'log', and it is non-public, .. :)
+  (: except if method name starts with 'log', or has 'Log' in the name, and it is non-public, .. :)
```

Comment:
comment can be: "except if method name contains log, trace, debug or info and is non-public"

Reply:
indeed